src/python/pants/backend/pants_info/list_target_types.py

```diff
@@ -31,7 +31,7 @@
 
 
 class TargetTypesSubsystem(LineOriented, GoalSubsystem):
-    """List all the registered target types, including custom plugin types."""
+    """List all registered target types."""
```

Comment:
Plugins are the basic way Pants works, there's no need to qualify. This might make the reader think "hm, where are plugins *not* included"?